go.sum

```diff
@@ -4,3 +4,5 @@ golang.org/x/sys v0.0.0-20200107162124-548cf772de50 h1:YvQ10rzcqWXLlJZ3XCUoO25sa
 golang.org/x/sys v0.0.0-20200107162124-548cf772de50/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
 golang.org/x/sys v0.0.0-20200116001909-b77594299b42 h1:vEOn+mP2zCOVzKckCZy6YsCtDblrpj/w7B9nxGNELpg=
 golang.org/x/sys v0.0.0-20200116001909-b77594299b42/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
+golang.org/x/sys v0.0.0-20200121082415-34d275377bf9 h1:N19i1HjUnR7TF7rMt8O4p3dLvqvmYyzB6ifMFmrbY50=
```

Comment:
It looks like @renovate-bot doesn't run `go mod tidy` after updating the dependency, leaving possibly unused old version hashes in `go.sum`.

Reply:
This seems to be a configuration option in `renovate.json`, see https://docs.renovatebot.com/configuration-options/#postupdateoptions

I sent #50 to add this option.

Reply:
OMG, I didn't know that renovate-bot does not run tidy... Thanks.